src/QsCompiler/Transformations/CallGraph.cs

```diff
@@ -199,8 +378,15 @@ public ImmutableArray<CallGraphEdge> InsertEdge(ImmutableArray<CallGraphEdge> ed
         /// </summary>
         public static CallGraphEdge CombinePathIntoSingleEdge(CallGraphNode targetNode, params CallGraphEdge[] edges)
         {
-            if (targetNode == null) throw new ArgumentNullException(nameof(targetNode));
-            if (edges == null || edges.Any(e => e == null)) throw new ArgumentNullException(nameof(edges));
+            if (targetNode == null)
+            {
+                throw new ArgumentNullException(nameof(targetNode));
+            }
+
```

Comment:
I think the blank line after closing braces is optional, right? (Just checking to make sure StyleCop is configured correctly.)

Reply:
It seems to be, the black line was automatically inserted when I automatically resolved the styling issues. When I remove the space between it looks at a glace like an `else if` rather that two unrelated `if` statements, so I prefer the blank line here.